libkbfs/folder_block_ops.go

```diff
@@ -960,7 +960,7 @@ func (fbo *folderBlockOps) nowUnixNano() int64 {
 // be the new DirEntry, and a local block cache. It also modifies md,
 // which must be a copy.
```

Comment:
Ah, yes, that's still true; all MD-modifying ops are done on the result of
head.MakeSuccessor(), which returns a (modified) copy. Probably worth
leaving in, in case someone else (me in 6 months) gets tempted to change it
back to ReadOnlyRootMetadata again.

On Thu, Jul 28, 2016 at 5:56 PM, John Zila <notifications@github.com> wrote:

> In libkbfs/folder_block_ops.go
> <https://github.com/keybase/kbfs/pull/200#discussion_r72728670>:
>
> > @@ -960,7 +960,7 @@ func (fbo *folderBlockOps) nowUnixNano() int64 {
> >  // be the new DirEntry, and a local block cache. It also modifies md,
> >  // which must be a copy.
>
> It needing to be a copy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/kbfs/pull/200/files/421184a1817399abc6b3478b8c097fd4254c692c#r72728670>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMp-Xsb2olAOv4Kzm6Dl3JkMJ6W7bRcgks5qaU_JgaJpZM4JWvBd>
> .
>